crates/ruff_python_formatter/README.md

```diff
@@ -323,3 +323,188 @@ await [1, 2, 3]
 This is more consistent to the formatting of other awaited expressions: Ruff and Black both
 remove parentheses around, e.g., `await (1)`, only retaining them when syntactically required,
 as in, e.g., `await (x := 1)`.
+
+### Line breaks between implicitly concatenated string attributes are not preserved ([#7052](https://github.com/astral-sh/ruff/issues/7052))
+
+Black preserves line breaks between implicitly concatenated string attributes.
+
+```python
+print(
```

Comment:
We should try to minimize all the examples to make them easier to understand:
* Remove unnecessary arguments
* Replace complex-expressions with simple expressions 
* Remove code that's unnecessary for the example